stable/raccoon/README.md

```diff
@@ -0,0 +1,97 @@
+# Raccoon
+
+A [helm](https://helm.sh/) chart for [Raccoon](https://github.com/odpf/raccoon)
+
+## Introduction
+
+This chart can be used for deploying [Raccoon](https://github.com/odpf/raccoon) on Kubernetes using helm
+
+## Prerequisites
+
+Install [helm3](https://helm.sh/docs/intro/install/#helm)
```

Comment:
Please provide other pre-requisites and explain assumptions such as kube cluster is running, appropriate kube context, namespace is set up etc., or provide reference to using `minikube` to test the charts.